addons/website_forum/static/src/js/website_forum.js

```diff
@@ -253,12 +253,11 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             // translation, to fix in the appropriate version
             notifOptions.message = `${karma} ${_t("karma is required to perform this action. ")}`;
             if (forumID) {
-                notifOptions.messageIsHtml = true;
                 const linkLabel = _.escape(_t("Read the guidelines to know how to gain karma."));
-                notifOptions.message = `
+                notifOptions.message = owl.markup(`
                     ${_.escape(notifOptions.message)}<br/>
                     <a class="alert-link" href="/forum/${forumID}/faq">${linkLabel}</a>
-                `;
+                `);
```

Comment:
This would benefit a lot from `owl.markup` being a template tag (properly escaping injected contents), but alternatively maybe this could use the `Markup` from web.utils instead? as I understand it *should* be compatible with `owl.markup` e.g.
```js
const linkLabel = _t("Read the guidelines to know how to gain karma.");
notifOptions.message = Markup`${notifOptions.message} <a class="alert-link" href="/forum/${forumID}/faq">${linkLabel}</a>`;